### ISSUE TITLE
Team 1: Implement get_ratios_of_comments function

### ISSUE THREAD

**Issue by mariakimheinert** [GatorGauge/GatorGauge#53]
`get_ratios_of_comments` should take in a list of strings that are Java programs and return a dict that contains the average ratios of singleline, multiline, and Javadoc comments to Java source code among all the strings. It should return something that looks like:
```
ratios_of_comments = {"singleline": 0.11, "multiline": 0.45, "javadoc": 0.32}
```
The averages should be kept to 2 decimal places.

**Comment by mariakimheinert** [comment]
@shafferz how is this feature going?

**Comment by delgrecoj** [comment]
Implemented in pull request #86.
